packages/amplication-client/src/util/truncatedWithEllipsis.ts

```diff
@@ -0,0 +1,8 @@
+export const truncateWithEllipsis = (
+  input: string | undefined,
+  limit: number,
+  defaultValue: string
+) => {
+  if (!input) return defaultValue;
+  return input.length > limit ? `${input.substring(0, limit)}...` : input;
+};
```

Comment:
You should be able to do it with just CSS.

Reply:
How? and should I do it? I fixed the places that used this function with `useMemo()`